docs/core-concepts/components.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Karmada Components
+---
+
+This document provides an overview of the components required for a fully functional and operational Karmada setup.
+
+![components](../resources/general/components.png)
+
+## Essential Components
+
+### karmada-apiserver
+
+This component uses the official image of Kubernetes and plays a crucial role in handling API requests,
+
+facilitating communication with the karmada-aggregated-apiserver, various controllers, karmada-scheduler,
+
+karmada-webhook, and persisting data in etcd.
```

Comment:
```suggestion
This component uses the official image of Kubernetes and plays a crucial role in handling API requests,
facilitating communication with the karmada-aggregated-apiserver, various controllers, karmada-scheduler,
karmada-webhook, and persisting data in etcd.
```
Do not add any blank line if it is a para or a sentence.

Reply:
Sorry, the blank line is automatic generated by my editor `typora`, I didn't pay attention to this. I'll fix it.